authentication/src/Login.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from "react";
+
+export default class Login extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      username: "",
+      password: ""
+    };
+  }
+
+  input = e => {
+    console.log(
+      "%c input has been triggered!",
+      "font-size:16px; color: green;"
+    );
+    this.setState({
+      [e.target.name]: e.target.value
+    });
+  };
+
+  clearInput = e => {
+    console.log("%c clearInput triggered!", "font-size:16px; color: violet");
+    e.preventDefault();
+    this.setState({
+      username: "",
+      password: ""
+    });
+  };
+
+  render() {
+    return (
+      <form onSubmit={() => this.clearInput()}>
```

Comment:
if you are not passing any parameters to the function you can just name it instead of making it an anonymous function...onSubmit={this.clearInput}